rollouts/controllers/rollout_controller.go

```diff
@@ -433,6 +428,50 @@ func (r *RolloutReconciler) computeTargets(ctx context.Context,
 	return targets, nil
 }
 
+func (r *RolloutReconciler) getWaveTargets(ctx context.Context, rollout *gitopsv1alpha1.Rollout, allTargets *Targets,
```

Comment:
these two function names `getWaveTargets` `getTargetsForWave` confused me a for a bit :)

Reply:
True. With the latest code I was able to eliminate the `getTargetsForWave` function.